newton/examples/example_anymal_c_walk_physx_policy.py

```
@@ -0,0 +1,256 @@
+
+###########################################################################
+# Example Anymal C walk
+#
+# Shows how to control Anymal C with a policy pretrained in physx.
+#
+# Example usage:
+# uv run --extra cu12 newton/examples/example_anymal_c_walk_physx_policy.py
+#
+###########################################################################
+
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)